package.json

```diff
@@ -31,9 +31,9 @@
     "getShortcuts": "ts-node ./scripts/getShortcuts.ts",
     "triggerShortcut": "ts-node ./scripts/triggerShortcut.ts",
     "start": "ts-node ./scripts/start.ts",
-    "deploy:staging": "gcloud beta functions deploy --gen2 --project=celo-mobile-alfajores --region=us-central1 --runtime=nodejs20 --env-vars-file=src/api/staging.yaml --update-labels \"valora-repo=hooks,commit-sha=$(git rev-parse HEAD)\"",
+    "deploy:staging": "gcloud beta functions deploy --gen2 --concurrency=80 --project=celo-mobile-alfajores --region=us-central1 --runtime=nodejs20 --env-vars-file=src/api/staging.yaml --update-labels \"valora-repo=hooks,commit-sha=$(git rev-parse HEAD)\"",
     "deploy:staging:http": "yarn deploy:staging --trigger-http --allow-unauthenticated",
-    "deploy:production": "gcloud beta functions deploy --gen2 --memory=512MB --project=celo-mobile-mainnet --region=us-central1 --runtime=nodejs20 --env-vars-file=src/api/production.yaml --update-labels \"valora-repo=hooks,commit-sha=$(git rev-parse HEAD)\" --set-secrets=NETWORK_ID_TO_RPC_URL=hooks-rpc-urls:latest",
+    "deploy:production": "gcloud beta functions deploy --gen2 --concurrency=80 --memory=512MB --project=celo-mobile-mainnet --region=us-central1 --runtime=nodejs20 --env-vars-file=src/api/production.yaml --update-labels \"valora-repo=hooks,commit-sha=$(git rev-parse HEAD)\" --set-secrets=NETWORK_ID_TO_RPC_URL=hooks-rpc-urls:latest",
```

Comment:
I manually bumped max instances from 100 to 200 on the console during the traffic spike, should we set it here to revert it back to 100, or maybe clear it? Is there a reason to have limit at all?